pkg/client/restclient/client.go

```diff
@@ -111,6 +111,10 @@ func readExpBackoffConfig() BackoffManager {
 			time.Duration(backoffDurationInt)*time.Second)}
 }
 
+func (c *RESTClient) GetRateLimiter() flowcontrol.RateLimiter {
```

Comment:
Right - it's not, originally I was doing something in this function, but in current state I can just remove it.